WiserTaskScheduler/WiserTaskScheduler/Program.cs

```diff
@@ -80,19 +81,30 @@ private static void ConfigureHostedServices(IServiceCollection services)
             services.AddHostedService<CleanupWorker>();
         }
 
-        private static void ConfigureWtsServices(IServiceCollection services)
+        private static void ConfigureWtsServices(IServiceCollection services, HostBuilderContext hostContext)
         {
             services.AddScoped<ConfigurationsWorker>();
             services.AddScoped<IDatabaseConnection, MySqlDatabaseConnection>();
-            services.AddScoped<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
             services.AddScoped<IObjectsService, ObjectsService>();
             services.AddScoped<IDatabaseHelpersService, MySqlDatabaseHelpersService>();
             services.AddScoped<IStringReplacementsService, StringReplacementsService>();
             services.AddScoped<ILanguagesService, LanguagesService>();
             services.AddScoped<IAccountsService, AccountsService>();
             services.AddScoped<IBranchesService, BranchesService>();
             services.AddScoped<IRolesService, RolesService>();
-
+            
+            // get SlackAccessToken & check if it is there 
+            var slackToken = hostContext.Configuration.GetSection("Wts").GetSection("SlackSettings").GetValue<string>("SlackAccessToken");
+            if (slackToken != String.Empty ){
+#if DEBUG
+            services.AddSingleton(new SlackEndpointConfiguration());
+#else
+            services.AddSingleton(new SlackEndpointConfiguration().UseSigningSecret(signingSecret));
+#endif
+            services.AddSlackNet(c => c.UseApiToken(slackToken));
+            }
```

Comment:
Graag de opmaak van de code aanpassen zodat die hetzelfde is als de rest van de code. Dus accolade op eigen regel, geen spatie tussen `String.Empty` en `)` en de code in de `if` inspringen naar rechts.

Daarnaast graag `slackToken != String.Empty` veranderen naar `!String.IsNullOrWhiteSpace(slackToken)`, zodat dit ook goed gaat als de waarde `null` is of iemand alleen een spatie invult.